timesketch/lib/analyzers/utils_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for analysis utils."""
+
+from __future__ import unicode_literals
+
+from timesketch.lib.testlib import BaseTest
+from timesketch.lib.analyzers import utils
+
+
+class TestAnalyzerUtils(BaseTest):
+    """Tests the functionality of the utilities."""
+
+    def __init__(self, *args, **kwargs):
+        super(TestAnalyzerUtils, self).__init__(*args, **kwargs)
```

Comment:
why even define __init__ when you are just calling super there....

Reply:
Ah, this is the scaffolder that adds this. Also I want to run the init on the BaseTestClass here.